xyz-hub-test/src/test/java/com/here/xyz/hub/rest/jobs/JobApiExportVersionIT.java

```diff
@@ -0,0 +1,308 @@
+                        .withProperties(new Properties().with("group", "movedFromEmpty")
+                                                        .with("vflag", "newest")),
+                AuthProfile.ACCESS_OWNER_1_ADMIN
+        );
+        postFeature(getScopedSpaceId(testVersionedSpaceId1Ext,scope), newFeature()
+                        .withId("id002")
+                        .withGeometry(new Point().withCoordinates(new PointCoordinates( 8.6199615,50.020093))) // htile 122001322012 , 23600774
+                        .withProperties(new Properties().with("group", "deltaonly")
+                                                        .with("vflag", "oldest")),
+                AuthProfile.ACCESS_OWNER_1_ADMIN
+        );
+        postFeature(getScopedSpaceId(testVersionedSpaceId1Ext,scope), newFeature()
+                        .withId("id002")
+                        .withGeometry(new Point().withCoordinates(new PointCoordinates( 8.6199615,50.020093))) // htile 122001322012 , 23600774
+                        .withProperties(new Properties().with("group", "deltaonly")
+                                                        .with("vflag", "inbetween")),
+                AuthProfile.ACCESS_OWNER_1_ADMIN
+        );
+        postFeature(getScopedSpaceId(testVersionedSpaceId1Ext,scope), newFeature()
+                        .withId("id002")
+                        .withGeometry(new Point().withCoordinates(new PointCoordinates( 8.6199615,50.020093))) // htile 122001322012 , 23600774
+                        .withProperties(new Properties().with("group", "deltaonly")
+                                                        .with("vflag", "newest")),
+                AuthProfile.ACCESS_OWNER_1_ADMIN
+        );
+        
+        deleteFeature(getScopedSpaceId(testVersionedSpaceId1Ext,scope), "id003");  // feature in base got deleted in delta
+
+        deleteAllJobsOnSpace(getScopedSpaceId(testVersionedSpaceId1, scope));
+        deleteAllJobsOnSpace(getScopedSpaceId(testVersionedSpaceId1Ext, scope));
+    }
+
+    @AfterClass
+    public static void clean(){
+
+        deleteAllJobsOnSpace(getScopedSpaceId(testVersionedSpaceId1, scope));
+        deleteAllJobsOnSpace(getScopedSpaceId(testVersionedSpaceId1Ext, scope));
+
+        removeSpace(getScopedSpaceId(testVersionedSpaceId1, scope));
+        removeSpace(getScopedSpaceId(testVersionedSpaceId1Ext, scope));
+    }
+
+    @Test
+    public void compositeL1Export_id_partitionidFcB64() throws Exception {
+// export by ID
+        Export.ExportTarget exportTarget = new Export.ExportTarget()
+                .withType(Export.ExportTarget.Type.VML)
+                .withTargetId(testVersionedSpaceId1Ext+":dummy");
+
+        /** Create job */
+        Export job =  buildTestJob(testExportJobId, null, exportTarget, Job.CSVFormat.PARTITIONID_FC_B64);
+        job.addParam("skipTrigger", true);
```

Comment:
skipTrigger param should be set inside `#buildTestJob()` as it's needed for all tests.